emily/handler/src/common/error.rs

```diff
@@ -172,7 +194,13 @@ impl From<SdkError<BatchWriteItemError>> for Error {
 }
 impl From<SdkError<UpdateItemError>> for Error {
     fn from(err: SdkError<UpdateItemError>) -> Self {
-        Error::Debug(format!("SdkError<UpdateItemError> - {err:?}"))
+        match err.into_service_error() {
+            // Note, this assumes that any conditional check that fails fails because
+            // there's a version conflict. This isn't necessarily true but is a good
+            // simplifying assumption.
```

Comment:
Do we know of any cases where this isn't the case?

Reply:
If you make any kind of conditional check this would fail and it would say it's a version conflict.